Source/Scene/Cesium3DTileset.js

```diff
@@ -442,26 +395,25 @@ define([
             var k;
             var additiveRefinment = (t.refine === Cesium3DTileRefine.ADD);
```

Comment:
Missing 'e' in variable name, `additiveRefinment` should be `additiveRefinement`